TheGiftpireStrikesBack/TheGiftpireStrikesBack/Contraband.swift

```diff
@@ -37,10 +37,19 @@ struct Wine: Identifiable {
     
 }
 
-enum Gift {
-    
+enum Gift: Identifiable {
     #warning("The Empire doesn't like unidentified contraband, to get through the check points we need to make `Gift` Identifable!")
+    var id: String {
```

Comment:
Did you check out the tests I also pushed to the repo? :) I think the Empire will smell something here when they see the mismatching ID's.